README.md

```diff
@@ -4,47 +4,70 @@
 uftrace
 =======
 
-The uftrace tool is to trace and analyze execution of a program
-written in C/C++.  It was heavily inspired by the ftrace framework
-of the Linux kernel (especially function graph tracer) and supports
-userspace programs.  It supports various kind of commands and filters
-to help analysis of the program execution and performance.
+uftrace traces, records and analyzes the execution of programs written in
+C, C++ and Rust. It was heavily inspired by the ftrace framework of the
+Linux kernel (especially the function graph tracer) and supports userspace
+programs, and with elevated privileges, also kernel function call graphs.
+
+It supports many commands and filters such as filtering by function call
+duration for analysis of program execution and performance.
 
 ![uftrace-live-demo](doc/uftrace-live-demo.gif)
 
  * Homepage: https://github.com/namhyung/uftrace
  * Tutorial: https://github.com/namhyung/uftrace/wiki/Tutorial
  * Chat: https://gitter.im/uftrace/uftrace
  * Mailing list: [uftrace@googlegroups.com](https://groups.google.com/forum/#!forum/uftrace)
+ * Lightning talk: https://youtu.be/LNav5qvyK7I
 
 
 Features
 ========
 
-It traces each function in the executable and shows time duration.  It
-can also trace external library calls - but usually entry and exit are
-supported.  Optionally it's possible to trace other (nested) external
-library calls and/or internal function calls in the library call.
-
-It can show detailed execution flow at function level, and report which
-function has the highest overhead.  And it also shows various information
-related the execution environment.
+uftrace traces each function in the executable and shows time durations.
+By hooking into the procedure linkage tables of shared libraries, it can
+also trace shared library calls like the `ltrace` tool, but with a nicer,
+more readable and colored output format which even includes resolving of
+C FILE pointers for stdin, stdout and stderr. Example with nested library
+calls:
+```py
```

Comment:
py ?

Reply:
I think it's to make the uftrace output prettier.